src/components/Timer/Timer.scss

```diff
@@ -84,48 +83,64 @@ $timer__short-actions-tooltip-height: 36px;
   justify-content: center;
   align-items: center;
 }
-.short-actions__button-wrapper > button > svg {
+.short-action__button > svg {
+  position: absolute;
   height: $icon--small;
   width: $icon--small;
 }
-.short-actions__button-wrapper > button:hover {
+.short-action__button--ready > .short-action__cancel-icon,
+.short-action__button--unready > .short-action__cancel-icon {
+  visibility: hidden;
+}
+.short-action__button--ready:hover > .short-action__check-icon {
+  visibility: hidden;
+}
+.short-action__button--ready:hover > .short-action__cancel-icon {
+  visibility: visible;
+}
+.short-action__button > .short-action__check-icon {
+  height: $timer__short-actions-height;
+  width: $timer__short-actions-height;
```

Comment:
personally, I like nesting selectors like these in one big statement like this
```scss
.selector {
  .child {
    /* ... */
  }
  :&hover {
    /* ... */
    .child {
       /* ... */ 
     }
  }
}
```
because I get a better overview of what is affected when  
But of course, your solution is not wrong or anything, just preference

Reply:
I don't like nesting that much 😢